packages/components/src/custom-select-control/test/index.tsx

```diff
@@ -716,7 +720,7 @@ describe( 'Type checking', () => {
 
 		const onChange = (): void => {};
 
-		<UncontrolledCustomSelectControl
+		<_CustomSelectControl
```

Comment:
Interestingly, changing this to the thinly wrapped version (`UncontrolledCustomSelectControl`) started changing some results to these static type tests (which, granted, are testing some advanced type inference logic). I guess `React.ComponentProps<>` can warp the types a bit, much like some other type utils (e.g. `Omit<>`).